client/my-sites/plan-features/style.scss

```diff
@@ -139,6 +154,46 @@ $plan-features-sidebar-width: 272px;
 	&.is-selected {
 		display: table-cell;
 	}
+	&.is-empty-feature-item {
+		background: #f6f6f6;
+	}
+
+	.is-no-tabs & {
+		border-right: 0;
+		&:nth-child( 9 ) {
```

Comment:
Does it do anything? Are we going to have 9 plans?

Reply:
I added empty TDs to implement the cellspacing in the design. There's space between the active and the overlayed plans. After this change we have 9 cells and not 5 or 3.